marlowe-dashboard-client/src/Template/Types.purs

```diff
@@ -11,31 +11,20 @@ import Analytics (class IsEvent, defaultEvent)
 import Data.BigInteger (BigInteger)
 import Data.Map (Map)
 import Data.Maybe (Maybe(..))
-import Marlowe.Extended (Contract, IntegerTemplateType, TemplateContent)
+import Marlowe.Extended (Contract, IntegerTemplateType, MetaData, TemplateContent, ContractTemplate)
+import Marlowe.Extended (MetaData) as ExtendedREs
 import Marlowe.Semantics (TokenName)
 
+type Template
+  = ContractTemplate
```

Comment:
why bother?

Reply:
It was to avoid disturbing the dashboard code (preserving naming)